app/popup/popup.html

```diff
@@ -203,6 +204,61 @@ <h3><strong id="statusCode"></strong></h3>
               Disable trace
             </button>
           </div>
+          <div class="tab-pane" id="testscookies">
+
+            <table width="100%" cellspacing="1" cellpadding="5" id="parent-list">
```

Comment:
I think that `parent-list` is not explicit. Can you rename it `feature-flag-table` ?